### PR TITLE
fix: decommission apps templateeditor and extract

### PR DESCRIPTION
**Pull request by DeepDiver1975**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: core/Migrations/Version20240131080456.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+/**
+ * Auto-generated migration step: Please modify to your needs!
+ */
+class Version20240131080456 implements ISimpleMigration {
+
+    /**
+     * @param IOutput $out
+     */
+    public function run(IOutput $out) {
```
Comment (phil-davis) [inline core/Migrations/Version20240131080456.php:15]: https://drone.owncloud.com/owncloud/core/39690/4/7
code-style is complaining about the indent of various lines.

File: core/Migrations/Version20240131080456.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?php
+namespace OC\Migrations;
+
+use OCP\Migration\ISimpleMigration;
+use OCP\Migration\IOutput;
+
+/**
+ * Auto-generated migration step: Please modify to your needs!
```
Comment (phil-davis) [inline core/Migrations/Version20240131080456.php:8]: Make the comment more relevant? Or delete it?

**Review by phil-davis — APPROVED** [review]
LGTM

**Comment by DeepDiver1975** [comment]
failed acceptance tests are due to timeout afaik ....

**Comment by phil-davis** [comment]
> failed acceptance tests are due to timeout afaik ....

agree, but I restarted CI anyway, to be sure that all pipelines will pass.

**Comment by DeepDiver1975** [comment]
THX

**Comment by prsnbrg** [comment]
@DeepDiver1975 @phil-davis 
Please revert this decision for the "extract" app. I have just published an update for the app in the Marketplace.

**Comment by DeepDiver1975** [comment]
@prsnbrg under which name did you publish the app on the marketplace? I could not find it. THX

**Comment by prsnbrg** [comment]
@DeepDiver1975 the update of the app is "published" under the same name as before (`extract`).
In the Marketplace the status of the app is "blocked". Can this be reverted or does the app have to be published under a new name?

**Comment by DeepDiver1975** [comment]
@IljaN

**Comment by DeepDiver1975** [comment]
Thx @prsnbrg 
Can you link the GitHub repo as well please?

**Comment by prsnbrg** [comment]
Sure!
The official repo of this app is: [https://github.com/PaulLereverend/NextcloudExtract](https://github.com/PaulLereverend/NextcloudExtract).
I create a fork for the owncloud app version (is aligned with Paul Lereverend): [https://github.com/prsnbrg/NextcloudExtract](https://github.com/prsnbrg/NextcloudExtract).

At the moment `info.xml` is linked to the repo from Paul Lereverend. My plan is to change this in the next few weeks with the next update.

**Comment by DeepDiver1975** [comment]
@prsnbrg I was rethinking the whole situation.
I suggest you publish the app under a different name.

Why? You have no access the the existing app in marketplace. And regaining this ist just too complicated.

What do you think? Thx a lot

**Comment by prsnbrg** [comment]
@DeepDiver1975 
Sorry for the confusion.
I have already published the first version of the app on Marketplace. However, since the app works perfectly for me and therefore the updates are somewhat neglected, which will not happen again in the near future.
It would be great if the app that was blocked in the Marketplace by ownCloud could be published again and the change in this PR reverted.

**Comment by IljaN** [comment]
@prsnbrg @DeepDiver1975 I unblocked the extract app

**Pull request merged**